sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -602,3 +783,4 @@ same output:
 * [Changing Organizations](https://docs.semaphoreci.com/article/29-changing-organizations)
 * [Pipeline YAML reference](https://docs.semaphoreci.com/article/50-pipeline-yaml)
 * [Dashboard YAML reference](https://docs.semaphoreci.com/article/60-dashboards-yaml-reference)
+* [Job YAML reference]()
```

Comment:
this needs something :)

Reply:
@markoa I know: https://github.com/semaphoreci/docs/pull/49 
The `Job YAML reference` page is not published yet.